Update up-and-running.md

ivanignacio opened the pull request:
# Why

Please describe the motivation for this PR, and link to relevant GitHub issues, forums posts, or feature requests.

I started learning expo and I've been installing it at home and work. Every time I create a new project I have to choose the template first and the project name after that. As a beginner, I can tell the information is not accurate, and even when this is not a major problem, I feel updating the documentation could help future developer starting using expo. 

# How

How did you build this feature or fix this bug and why?

I simply rewrote the paragraph so new developers can know what to expect

# Test Plan

Please describe how you tested this change and how a reviewer could reproduce your test, especially if this PR does not include automated tests! If possible, please also provide terminal output and/or screenshots demonstrating your test/reproduction.

N/A

ivanignacio committed:
Update up-and-running.md

Fixes the guide related with the first step when creating a new project.

Commit:
Merge pull request #1 from ivanignacio/ivanignacio-patch-1

Update up-and-running.md

AdamJNavarro commented:
Hey @ivanignacio,

Can you explain why you think these changes should be made?

Thanks,
Adam

ivanignacio commented:
Hi @AdamJNavarro , I just updated the post.

Thanks,
Ivan

Commit:
Update how-expo-works.md

docs/pages/versions/v31.0.0/workflow/how-expo-works.md
Removes an extra 'the' in a sentence

Commit:
Update how-expo-works.md

docs/pages/versions/unversioned/workflow/how-expo-works.md
Removes an extra 'the' in a sentence

Commit:
Update how-expo-works.md

Removes an extra 'the' in a sentence

sjchmiela commented:
"Up and running" page has been removed from recent versions of the docs.
